src/UI/Factory.php

```diff
@@ -336,4 +337,62 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers can be used if space is scarce to display secondary information such as a preview of
+	 *     some item or to offer limited user interaction.
```

Comment:
Thanks @atoedt, I change it accordingly! If there are other suggestions we are still able to update it.